widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -544,7 +544,8 @@ internal class ChatController(
             viewCallback?.backToCall()
         } else {
             backClickedListener?.onBackClicked()
-            Dependencies.getControllerFactory().destroyControllers()
+            Dependencies.getControllerFactory().destroyChatController()
+            Dependencies.getControllerFactory().destroyCallController()
```

Comment:
I am not 100% sure, but possibly you might want to also reset ActivityWatcherForCallVisualizer. Technically it is possible that it is displaying some Dialog in the integrator app when clearSession was called and it should then dismiss any pending dialogs.

Reply:
Honestly, I am not sure that resetting anything additionally won't brake something.
I would add it only in case we see the need (e.g. flow that needs it).
I haven't seen such cases so far.
So, if you don't mind, I will leave it as it is now.